### ISSUE TITLE
is there any reference paper for mycroft-precise ?

### ISSUE THREAD

**Issue by OrangeBaoWang** [MycroftAI/mycroft-precise#39]
Hi , I want to try the model on android, but there are some question  about the features that feed into net.
there are three features include mels、mfccs、speechpy_mfccs according to code in  vectorization.py. what's difference between them ?  how to calculate them ? is there any theory reference paper for mycroft-precise ?

**Comment by MatthewScholefield** [comment]
Hello, there's no reference paper for Precise since everything is rather simple and stripped down. It's all a series of experiments and those are some of the methods we have tried. Here's an explanation of each of them:
 - *Mfccs:* The default audio features used. It uses [sonopy](https://github.com/MycroftAI/sonopy) to calculate mfcc vectors. [Take a look at the step by step example here](https://github.com/MycroftAI/mycroft-precise/wiki/Research#audio-features).
 - *Mels:* The same as the previous audio feature extraction method, but without the last DCT. This makes the result spatially coherent along the feature axis (the bottom represents low sounds and the top represents high sounds). Basically, it works very similar to mfccs, but it can work with 2D convolutions. It looks like a condensed representation of a spectrogram. However, through experiments, I haven't found any architecture that works with this better than MFCCs.
 - *speechpy_mfccs*: Mfccs using a different library. Only for backwards compatibility.

TL;DR, you only need to focus on the `mfccs` option. You can find the parameters [here](https://github.com/MycroftAI/mycroft-precise/blob/dev/precise/params.py#L74) and the calculations [here](https://github.com/MycroftAI/sonopy/blob/master/sonopy.py#L74).
